aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +258,40 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Full-page form
+
+- Use full-page forms for create and edit flows, irrespective of the size of the form.
```

Comment:
Why?

We should be educating and explaining the rationale and why it makes for a better experience. Otherwise we are just dictating "because I said so." How do we go back and justify/rationalize this guidance and the decisions made?....